style.css

```diff
@@ -77,6 +50,62 @@ body {
     
 }
 
+.menu {
+    display: flex;
+    flex: 1; 
+    flex-direction: column;
+}
+
+#title-box-wrapper {
```

Comment:
We should only be styling classes, never ids

Reply:
!! Ok, I didn't know that, why is that it?

Reply:
should I change all the ids to classes?

Reply:
Generally, `ids` are used for selecting elements with javascript and `classes` are used for selecting elements with css -- when you style an element with css we never know if we are going to want to re-use that element later. If we do, we can just copy the element within our html and attach the same classes and it will look exactly the same. (even though you may think now you only have 1 play-button it's better to assume you don't know, and you should strive to **make** classes that you will use more than once)

That "have-more-than-one-ness" is why it is a `class` - a group of things that are alike in some way.

`ids` in contrast are unique, and required to be so.

Let's look at a beepbox example:

<img width="191" alt="Screen Shot 2019-06-07 at 7 20 54 AM" src="https://user-images.githubusercontent.com/8755912/59110898-e1035300-88f4-11e9-9f72-fa24d6027df6.png">

we could do it like this:
```
#button-play { background: grey; }
#button-back { background: grey; }
#button-forward { background: grey; }

<button id="button-play">play</button>
<button id="button-back">back</button>
<button id="button-forward">forward</button>
```

but look at all that repetition.. since we use ids, we cant just apply the same style to all 3 elements -- because ids must be unique to one element.

One step up from this structure would be:
```
.menu-button { background: grey; }
...
<button class="menu-button">play</button>
<button class="menu-button">back</button>
<button class="menu-button">forward</button>
```

There isn't a "right" way to do it, and there are a ton if different philosophies. But, you'll find that if you follow the rule of `classes-for-styling` and `ids-for-js` these kinds of cleanups popup a lot, and your code will tend to be a bit nicer